_markets/at/technical-operations/rollout-automation/templates/_variables.mdx

```diff
@@ -1 +1,64 @@
-Templates for Austrian CashBoxes only make use of the standard variables.
\ No newline at end of file
+| Variable                                    | Content                                                      |
+| ------------------------------------------- | ------------------------------------------------------------ |
+| `reference_scu_{type}_{key}_{value}_{attr}` | References the indicated attribute (`attr`) of an existing SCU with the given key/value pair (`key`/`value`) and the specified type (`type`). |
+| `reference_scu_{type}_{attr}` | References the indicated attribute (`attr`) of the first existing SCU and the specified type (`type`). |
+
+### Reference variables
+
+Reference variables allow you to reference values from other components. Their overall name structure is
+
+```
+reference_scu_{type}_{key}_{value}_{attr}
+
+reference_scu_{type}_{attr}
+```
+
+Where the four dynamic parts (denoted by curly brackets) specify which attribute of which component you exactly want to reference. In the second reference where the reference doesn't contain a key/value, the first existing SCU is selected.
+
+
+
+#### Type
+
+Indicates the SCU type you want to reference. Possible values for `type` are
+
+* `onlinescu`
+
+#### Key / Value
+
+Provides a key-value pair that you can use to filter for the SCU of the indicated SCU type. The scope is always your **account** and the **outlet** where you create the CashBox. You cannot reference SCUs from other outlets or accounts.
+
+
+
+#### Attribute
+
+The actual attribute you'd like to reference. Currently, `id` and `url` are supported.
+
+
+
+#### Example
+
+:::info Default field with shop-bought SCUs
+
+Any TSE bought via fiskaltrust.Shop or 'Rollout Management' can have an incrementing integer field `counter` assigned.
```

Comment:
SCU, the term TSE only exists in germany, doesn't exist in AT

Reply:
done